docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md

```diff
@@ -336,7 +291,6 @@ The following are the required permissions for installing the Amazon EBS CSI Dri
         "eks:AssociateIdentityProviderConfig",
         "eks:CreateAddon",
         "eks:DeleteAddon",
-        "eks:DescribeAddon",
```

Comment:
Please decide whether to keep this in place or remove all duplicate entries which are already in Main block:
```
"eks:DescribeCluster",
"iam:AttachRolePolicy",
"iam:CreateRole",
"iam:GetRole",
"iam:ListAttachedRolePolicies",
"iam:PassRole",
"iam:TagRole",
```

Reply:
Good catch, I will remove the duplicated entries.